CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+<!-- Note: Update the `Unreleased link` after adding a new release -->
+
+## [Unreleased](https://github.com/appsembler/tahoe-figures-plugins/compare/v0.1.1...HEAD)
+
+## [0.1.1](https://github.com/appsembler/tahoe-figures-plugins/compare/v0.1.0...v0.1.1) - 2022-08-17
+ - Use tahoe-sites package
+
+## [0.1.0] - 2022-02-03
+ - First release
```

Comment:
Thanks for making this file.


Changelog is in README. Please remove it from there.

Reply:
Done